sdk/all/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java

```diff
@@ -82,4 +85,15 @@ void testReconfigure() {
     assertThat(openTelemetry.getResource()).isEqualTo(resource);
     assertThat(openTelemetry.getClock()).isEqualTo(clock);
   }
+
+  @Test
+  void testTracerProviderAccess() {
+    OpenTelemetrySdk openTelemetry =
+        OpenTelemetrySdk.builder()
+            .setTracerProvider(tracerProvider)
+            .setTracerSdkManagement(tracerSdkManagement)
```

Comment:
What if I don't set `tracerSdkManagement` manually? Will this test fail?

Reply:
it will, because you haven't fully built a valid SDK.